doc/CMakeLists.txt

```diff
@@ -14,6 +14,14 @@ if(POLICY CMP0074)
   cmake_policy(SET CMP0074 NEW)
 endif()
 
+# If OPENVDB_DOXYGEN_HOUDINI is set, the Houdini documentation is also
+# built. You can provide this module a value for Houdini_VERSION which will
+# take into account any conditional compiled blocks. Whilst this doesn't
+# require you to build the Houdini plugin, building from the root CMake
+# with OPENVDB_BUILD_HOUDINI_PLUGIN or USE_HOUDINI will define this value.
```

Comment:
I would expect this value to only be set to ON if OPENVDB_BUILD_HOUDINI_PLUGIN=ON **and** OPENVDB_BUILD_DOCS=ON.

Reply:
The point of this option is so you don't have to have `OPENVDB_BUILD_HOUDINI_PLUGIN=ON` set, as building the Houdini plugin isn't connected to these docs

Reply:
Understood. I'm saying that if you _do_ have OPENVDB_BUILD_HOUDINI_PLUGIN=ON set, it shouldn't build the Houdini docs _unless_ you also have OPENVDB_BUILD_DOCS=ON set.

Reply:
Ah gotchya - but that's the case anyway as unless OPENVDB_BUILD_DOCS is set this CMakeLists file never gets added. So no docs are built unless OPENVDB_BUILD_DOCS=ON, where by all docs are built by default irrespective of the OPENVDB_BUILD_HOUDINI_PLUGIN option

Reply:
Sure, it's perhaps just worth adding a small caveat to make that explicit in this comment. Although it's convention for the project, the wording:

> building from the root CMake with OPENVDB_BUILD_HOUDINI_PLUGIN or USE_HOUDINI will define this value

introduces some uncertainty that could still require tracing back to the root CMakelists to find out that this file is only in use if OPENVDB_BUILD_DOCS is set. 

It's a minor thing though, I'll mark the PR as approved anyway and you can decide what to do here.